notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1986 @@
+{
```

Comment:
<p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Fixed.</p>

Reply:
<p>we can reuse -&gt; you can reuse</p><p>This could help: 1. reducing the variation... 2. reducing training... -&gt; This can reduce the variation of the output model or the training cost. </p><p> </p><p>Since the code demonstrates how to load the result programmatically, I would change the last line to the following:</p><p>"The existing architecture search result is stored in the <code>tuning_result_output</code> output of the <code>automl-forecasting-stage-1-tuner</code> component. You can load it programmatically with the API."</p>

Reply:
<p>Fixed.</p>